tests/playwright/README.md

```diff
@@ -0,0 +1,136 @@
+# Podman Desktop Playwright Tests
+
+Testing Framework dedicated to a Podman Desktop and its extensions.
+
+## How to build locally
+
+1. Node 20 (ideal is to use `nvm`)
+2. Install local dependencies: `npm install`
+3. Build: `npm run build`
+4. Create Local package `npm run package`
+5. Connect to the repository where you want to write your tests
+
+## Usage
```

Comment:
it's not clear in usage if it's when using in this repository or when wanting to embed/use in a 3rd party extension repository

Reply:
it has been flagged as resolved but looks like there was no reply

AFAIK the README will be published on npmjs so I don't know why it says to clone podman-desktop repository

it should start at step 7

Reply:
is there a reason why it's marked as resolved ?